project/epi/serializers.py

```diff
@@ -81,6 +81,8 @@ class Meta:
 class SimpleExposureSerializer(serializers.ModelSerializer):
     url = serializers.CharField(source='get_absolute_url', read_only=True)
     metric_units = DoseUnitsSerializer()
+    variance_type = serializers.CharField(source='get_variance_type_display', read_only=True)
+    estimate_type = serializers.CharField(source='get_estimate_type_display', read_only=True)
```

Comment:
@JoshAddington Had to add these fields to the `SimpleExposureSerializer`, since they were being reported as integer values in Excel export